packages/emotion/js/Css.ml

```diff
@@ -7,12 +7,12 @@ include Css_Legacy_Core.Make (struct
   type renderer = Js.Json.t
 
   external injectRaw : string -> unit = "injectGlobal"
-  [@@mel.module "@emotion/css"] [@@bs.module "@emotion/css"]
+  [@@mel.module "@emotion/css"] [@@mel.module "@emotion/css"]
```

Comment:
That's breaking ReScript users, I will jump into this PR to make it work since it needs to duplicate this file and it's a little messy right now

Reply:
Do you think cppo would help here?

Reply:
Oh, also cppo!